test/inductor/test_torchinductor_opinfo.py

```diff
@@ -520,6 +521,7 @@ class TestInductorOpInfo(TestCase):
     )  # inductor kernels failing this test intermittently
     @skipCUDAIf(not HAS_CUDA, "Skipped! Triton not found")
     @skipCPUIf(not HAS_CPU, "Skipped! Supported CPU compiler not found")
+    @unittest.skipIf(TEST_WITH_ASAN, "Skipped under ASAN")
```

Comment:
Any reason for this skipping?

Reply:
I think it's failing now because we run constant folding on empty kernels, which breaks ASAN. If you look around there are a number of ASAN skips. I don't think it's providing a lot of value here..